osv/ecosystems/_ecosystems.py

```diff
@@ -30,6 +31,7 @@
     'Packagist': Packagist(),
     'Pub': Pub(),
     'PyPI': PyPI(),
+    'R': R(),
```

Comment:
This needs to match the ecosystems defined in the schema, so it should be `CRAN` (and also `Bioconductor`) based on https://github.com/ossf/osv-schema/pull/176